lib/utils.js

```diff
@@ -16,15 +17,37 @@ module.exports.extractKVBindings = (setFlagValues) => {
     const value = kvComponents.slice(1).join('=')
 
     if (!bindings[variable]) {
-      bindings[variable] = new KeyValueStore()
+      bindings[variable] = new KeyValueStore(filepaths[variable])
     }
 
     bindings[variable].put(key, value)
   }
 
+  for (const [variable, path] of Object.entries(filepaths)) {
+    bindings[variable] = new KeyValueStore(path)
+  }
+
   return bindings
 }
 
+const extractKVPaths = (kvFileFlags) => {
+  const paths = {}
+
+  if (kvFileFlags && kvFileFlags.length > 0) {
```

Comment:
length check is redundant. Furthermore, I'm a fan of returning early if no work needs to be done. Rather than wrapping the for loop in a conditional, let's put something like the below before the for loop:
```
if (!kvFileFlags) { return paths }
```

Reply:
Yes, good call, resolved!